test/functional/apps/dashboard/time_zones.js

```diff
@@ -65,7 +65,7 @@ export default function ({ getService, getPageObjects }) {
     it('Changing timezone changes dashboard timestamp and shows the same data', async () => {
       await PageObjects.settings.navigateTo();
       await PageObjects.settings.clickKibanaSettings();
-      await PageObjects.settings.setAdvancedSettingsSelect('dateFormat:tz', 'EST');
+      await PageObjects.settings.setAdvancedSettingsSelect('dateFormat:tz', 'Etc/GMT+5');
```

Comment:
:information_source: This test did obviously not test what it wanted to test, since it used a timezone, that would not actually allow requests. I assume the test did pass, because it most likely did never refresh the page properly. I want to look into the test separately from this PR, and make it work properly, but for this PR I simply replaced the removed timezone by an equivalent still available time zone.